python/eks/cluster/app.py

```diff
@@ -0,0 +1,123 @@
+from aws_cdk import (
+    aws_autoscaling as autoscaling,
+    aws_ec2 as ec2,
+    aws_eks as eks,
+    aws_iam as iam,
+    core,
+    custom_resources as cr,
+)
+
+class Stack(core.Stack):
+
+    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        vpc = ec2.Vpc(
+            self,
+            "Vpc",
+            max_azs=3,
+            cidr="10.0.0.0/16",
+        )
+
+        # it's good to add these tags in case you may want to deploy a Ingress Controller
+        for public_subnet in vpc.public_subnets:
+            core.Tags.of(public_subnet).add("kubernetes.io/role/elb", "1")
+
+        for private_subnet in vpc.public_subnets:
+            core.Tags.of(private_subnet).add("kubernetes.io/role/internal-elb", "1")
```

Comment:
```suggestion
        for public_subnet in vpc.public_subnets:
            Tags.of(public_subnet).add("kubernetes.io/role/elb", "1")

        for private_subnet in vpc.public_subnets:
            Tags.of(private_subnet).add("kubernetes.io/role/internal-elb", "1")
```

Upgrade to V2

Reply:
don't we need to import `Tags` as well?

Reply:
Yep but that should already be imported on line 5